torch_ema/ema.py

```diff
@@ -60,16 +60,30 @@ def copy_to(self, parameters):
             if param.requires_grad:
                 param.data.copy_(s_param.data)
 
+    def store(self, parameters):
```

Comment:
Really nice! Could you also remove lines 57 and 59 (those that reference `self.collected_parameters` in `copy_to`)?

Other than that, I'd also suggest adding `self.collected_parameters = []` in `__init__()` and I'd be ready to merge.

Reply:
Of course.